src/components/Tooltip/Tooltip.tsx

```diff
@@ -205,7 +205,7 @@ export const Tooltip = ({
     }, [enterDelay]);
 
     const checkIfHovered = useCallback(
-        (event) => {
+        (event: any) => {
```

Comment:
```ts
      const checkIfHovered = useCallback(
        (event: MouseEvent) => {
            const hoveredElement = event.composedPath?.();
            const hoverSources = [triggerRefElement, triggerElementContainerRef, tooltipContainerRef];

            if (hoveredElement && hoverSources.some((el) => el.current && hoveredElement.includes(el.current))) {
                handleShowTooltipOnHover();
            }
        },
        [handleShowTooltipOnHover],
    );
```
You can do this

Reply:
Thanks for the suggestion! I just updated that one. 😄